core/server/worker/src/main/java/alluxio/worker/grpc/AbstractWriteHandler.java

```diff
@@ -53,6 +60,9 @@
   private static final Logger LOG = LoggerFactory.getLogger(AbstractWriteHandler.class);
 
   private final StreamObserver<WriteResponse> mResponseObserver;
+  private final SerializingExecutor mSerializingExecutor;
```

Comment:
can you add comments on these fields? I don't fully understand them.

Reply:
Done.